app/components/Capture.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import html2canvas from 'html2canvas';
+import IoAndroidCamera from 'react-icons/lib/io/android-camera';
+import ShareScreen from './ShareScreen';
+
+class Capture extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      img: '',
+      isOpen: false,
+    };
+    this.takeScreenshot = this.takeScreenshot.bind(this);
+    this.toggleModal = this.toggleModal.bind(this);
+  }
+
+  toggleModal = () => {
+    this.setState({
+      isOpen: !this.state.isOpen,
+    });
+  };
+
+  takeScreenshot() {
+    const h = this;
+    html2canvas(document.body).then(canvas => {
```

Comment:
don't have to capture the body but the report container, also create a new virtual element because there might be css changes needed to be made.

Reply:
I've updated the code to capture report container only.